core/controllers/classifier_test.py

```diff
@@ -61,8 +59,9 @@ def setUp(self) -> None:
             feconf.TESTS_DATA_DIR, 'string_classifier_test.yaml')
         with utils.open_file(yaml_path, 'r') as yaml_file:
             self.yaml_content = yaml_file.read()
-        self.signup(self.CURRICULUM_ADMIN_EMAIL, self.CURRICULUM_ADMIN_USERNAME)
-        self.signup('moderator@example.com', 'mod')
+        self.signup(
+            feconf.ADMIN_EMAIL_ADDRESS, self.CURRICULUM_ADMIN_USERNAME, True)
```

Comment:
Perhaps this should be self.ADMIN_USERNAME? Or do you want two separate users?

The whole point of naming these constants like this is to make it easy for whoever is reading the tests to identify the user, so mixing and matching makes things a bit muddy.

Reply:
I agree, The reason I did this is because we do not have self.ADMIN_USERNAME or feconf.ADMIN_USERNAME and that is why I used that.
Should I introduce a new variable in the test_utils.py file?

Reply:
That said, test_utils.py already has SUPER_ADMIN_EMAIL which I suppose can be used?

Reply:
I have updated setUp() with the username variable.
Actually we cannot because the mail will only be sent to the `feconf.ADMIN_EMAIL_ADDRESS`.